maas/plugins/galera_check.py

```diff
@@ -82,25 +86,35 @@ def print_metrics(replica_status):
            replica_status['wsrep_local_state_uuid'])
     metric('wsrep_local_state_comment', 'string',
            replica_status['wsrep_local_state_comment'])
+    metric('max_configured_connections', 'int64',
+           replica_status['max_connections'], 'connections')
+    metric('current_connections', 'int64',
+           replica_status['Threads_connected'], 'connections')
+    metric('max_seen_connections', 'int64',
+           replica_status['Max_used_connections'], 'connections')
+    metric('percentage_used_connections', 'int64',
```

Comment:
Is this metric required? The other two values are already being recorded and MaaS alarm language can be used to calculate percentages if required.